### DISCUSSION TITLE
Way to render menus in the body?

### DISCUSSION THREAD

**Discussion by httpete** [ariakit/ariakit#1498] [Questions] [answered]
Due to painful zIndexing issues, I want to render the menus down in the body element, like a Portal does (and have it all work the same) possible? Thanks

**Comment by diegohaz** [comment, marked as answer]
Yes, use the `portal` prop:
```jsx
<Menu portal>
```

**Reply by httpete** [reply]
This is fantastic, it works perfectly, and saved my day ! You the man @diegohaz . Looking forward to the new ariakit doc site.
